CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake")
 
 # put executables in the 'run' folder
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_SOURCE_DIR}/run)
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_SOURCE_DIR}/run)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_SOURCE_DIR}/run/lib)
```

Comment:
@jpfeltracco reverting this line will fix the test-python build problem. The reason that it was working for us is that the robocup.so file (which is placed in `run` was stale, and not removed via a make clean (which should be fixed), and the new file was going to `./run/lib`. I do agree that this is a better idea, but we need to be 100% sure that everything changes the path correctly when we do.